google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/JobSnippets.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * EDITING INSTRUCTIONS
+ * This file is referenced in Job's javadoc. Any change to this file should be reflected in
+ * Job's javadoc.
+ */
+
+package com.google.cloud.examples.bigquery.snippets;
+
+import com.google.cloud.bigquery.Job;
+import com.google.cloud.logging.LoggingOptions;
+import com.google.cloud.logging.Sink;
+import com.google.cloud.logging.SinkInfo;
+
+public class JobSnippets {
```

Comment:
I just ran locally, but I did not check it in since it deletes the other hand-written snippets. I'll add the others before removing the hand-written ones.